contents/handbook/people/team-structure/app-east.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Team App East
+sidebar: Handbook
+showTitle: true
+hideAnchor: true
+---
+
+## People
+
+- Marius Andra (Team lead)
+- Ben White (Engineer)
+- Paul D'Ambra (Engineer)
+- Michael Matloka (Engineer)
+- Emanuele Capparelli (Growth Engineer)
+
+## Mission
+
+Makers everywhere get better at building products because of PostHog
+
+## Q3 2022 Goals
+* **Objective 1:**
+    * Increase the number of successful organizations on PostHog
+* **Key Results:**
+    * Increase in [retention](https://app.posthog.com/insights/bj3-djOy): week 4 retention from the previous week goes from 21.5% to 30%.
+* **Rationale:**
+    * To nail self-serve we need every new organization that signs up to have a smooth and easy experience, from ingesting their first event to understanding how to use product analytics to inviting their team to share insights
+
+* **Objective 2:**
+    * Make it **easy to build** new features quickly that fit in with our design system.
+* **Key Results:**
+    * 50% reduction in custom and inline CSS styles. Exact metric TBD.
+* **Rationale:**
+    * PostHog engineers are slowed down with our legacy frontend practices.
```

Comment:
@clarkus we had a sync meeting and made changes here. 

For objective 1, we'd like to set up a sync meeting for early next week to brainstorm all the things we can build that can move this needle. One possible time is next Monday, by starting our team's standup 30min earlier. Is that too early?

For objective 2, the key result is still TBD, but our bet is that the less custom CSS we have around, the more people are actually using our design system as it's intended to be used. This also means that we rely less on hacky antd components/css, etc. It's not an ideal metric, but could be a close enough proxy. The objective is still to build a great design system, not purely to yeet css.

Any thoughts on these?